pkg/sql/gcjob/refresh_statuses.go

```diff
@@ -461,19 +461,6 @@ func refreshTenant(
 	}
 
 	tenID := details.Tenant.ID
-	// TODO(ssd): Once
-	// https://github.com/cockroachdb/cockroach/issues/92093 is
-	// done, we should be able to simply rely on the protected
-	// timestamp for the replication job.
-	jobActive, err := tenantHasActiveReplicationJob(ctx, execCfg, tenID)
```

Comment:
I think we can nuke `tenantHasActiveReplicationJob` as well.

We _may_ want to beef up the tests around this garbage collection waiting, but that doesn't need to happen in this PR.